diff --git a/node_modules/redux-flipper/lib/index.js b/node_modules/redux-flipper/lib/index.js
index 8cc534c..a3d6cea 100644
--- a/node_modules/redux-flipper/lib/index.js
+++ b/node_modules/redux-flipper/lib/index.js
@@ -2,6 +2,7 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 const react_native_flipper_1 = require("react-native-flipper");
 const dayjs = require("dayjs");
+const cycle = require('cycle');
 const defaultConfig = {
     resolveCyclic: false,
     actionsBlacklist: [],
@@ -18,7 +19,7 @@ const createStateForAction = (state, config) => {
 // To initiate initial state tree
 const createInitialAction = (store, config) => {
     const startTime = Date.now();
-    let initState = store.getState();
+    let initState = store.getState().toJS();
     if (config.resolveCyclic) {
         const cycle = require('cycle');
         initState = cycle.decycle(initState);
@@ -73,10 +74,10 @@ const createDebugger = (config = defaultConfig) => (store) => {
     return (next) => (action) => {
         var _a;
         let startTime = Date.now();
-        let before = store.getState();
+        let before = store.getState().toJS();
         let result = next(action);
         if (currentConnection) {
-            let after = store.getState();
+            let after = store.getState().toJS();
             let now = Date.now();
             let decycledAction = null;
             if (config.resolveCyclic) {
